google/cloud/security/inventory/pipeline_configs.yaml

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
nit: We need the LICENSE here too.
nit: I'd like to store default configs for the core-modules in /config/inventory.(conf|yaml) for example.